### PR TITLE
/{gen_release_notes.pl, install-deps.sh}: format notes to include dependency repo names

### PR DESCRIPTION
**Pull request by coffeegoddd**

### CODE REVIEW

File: gen_release_notes.pl [hunk 1]
```diff
@@ -131,20 +132,27 @@
...
-    if ($pr->{body}) {
-        my @lines = split (/\s*[\n\r]+\s*/, $pr->{body});
-        foreach my $line (@lines) {
-            print "  $line\n";
+print "# Merged PRs\n\n";
+foreach my $dep (@sorted_deps) {
+    my $repo_name = format_repo_name($dep);
+    print "## $repo_name\n\n";
+    my $dep_arr = $merged_prs_dict{$dep};
+    foreach my $pr (@$dep_arr) {
```
Comment (zachmu) [inline gen_release_notes.pl:149]: Does this die if there are no merged PRs for a particular dep?

Comment (zachmu) [reply gen_release_notes.pl:149]: I would just do 
`next unless $dep_arr` above

File: gen_release_notes.pl [hunk 2]
```diff
@@ -131,20 +132,27 @@
...
 
-    push @$merged_prs, @$merged_dep_prs;
+    $merged_prs_dict{$dep} = $merged_dep_prs;
     push @$closed_issues, @$closed_dep_issues;
 }
 
-print "# Merged PRs\n\n";
-foreach my $pr (@$merged_prs) {
-    print "* [$pr->{number}]($pr->{url}): $pr->{title}\n";
+my @sorted_deps = nsort keys %merged_prs_dict;
```
Comment (zachmu) [inline gen_release_notes.pl:139]: Do you need nsort here? Aren't these just strings (repo names)?

**Review by zachmu — APPROVED** [review]
LGTM

File: gen_release_notes.pl (hunk 2 above)
Comment (coffeegoddd) [reply gen_release_notes.pl:139]: they are strings, but im sorting them so the `dolt` changes print first, then gms, then vitess... it looked weird when vitess came before the dolt changes in the output

**Pull request merged**